### PR TITLE
Tests for Rotate and RotateAroundCenter, Skew, SkewAroundCenter

### PR DESCRIPTION
**Pull request by drott**
Includes adaptations for development branch of fontTools that has latest spec changes.

### CODE REVIEW

**Comment by drott** [comment]
Your review is appreciated, @anthrotype - not planning to merge to main just yet, but using this rotateAndSkewTests branch. We can also name the branch something else if we want to put other "after spec changes" tests here.

File: config/more_samples-glyf_colr_1.py [hunk 1]
```diff
@@ -372,6 +372,61 @@ def _extend_modes(gradient_format, extend_mode, accessor_char):
...
+        rotated_colr = {
+            "Format": ot.PaintFormat.PaintRotateAroundCenter,
+            "centerX": center_x,
+            "centerY": center_y,
+            "angle": angle,
+        }
+    else:
+        rotated_colr = {"Format": ot.PaintFormat.PaintRotate, "angle": angle}
+
+    rotated_colr = {**rotated_colr, **glyph_paint}
```
Comment (anthrotype) [inline config/more_samples-glyf_colr_1.py:402]: rotated_colr.update(glyph_paint)

File: config/more_samples-glyf_colr_1.py [hunk 2]
```diff
@@ -427,6 +427,64 @@ def _paint_rotate(angle, center_x, center_y, accessor_char):
...
+
+    if center_x or center_y:
+        skewed_colr["Format"] = ot.PaintFormat.PaintSkewAroundCenter
+        skewed_colr["centerX"] = center_x
+        skewed_colr["centerY"] = center_y
+
+    else:
+        skewed_colr["Format"] = ot.PaintFormat.PaintSkew
+
+    skewed_colr = {**skewed_colr, **glyph_paint}
```
Comment (anthrotype) [inline config/more_samples-glyf_colr_1.py:460]: skewed_colr.update(glyph_paint)

**Review by anthrotype — APPROVED** [review]
lgtm

yes let's hold this until updated fonttools is released with the changes

File: config/more_samples-glyf_colr_1.py (hunk 2 above)
Comment (drott) [reply config/more_samples-glyf_colr_1.py:460]: Done.

File: config/more_samples-glyf_colr_1.py (hunk 1 above)
Comment (drott) [reply config/more_samples-glyf_colr_1.py]: Done.

**Comment by drott** [comment]
Tested with released fonttools from main branch https://github.com/fonttools/fonttools/commit/f3a903093a4600f3d757bf9b22f387a561fcb584 - merging.

**Pull request merged**